commands.js

```diff
@@ -420,7 +422,10 @@ module.exports = function() {
     updatable = [RACE_COLUMN, SUBRACE_COLUMN, GENDER_COLUMN, CLASS_COLUMN, STATUS_COLUMN, NATIVE_COLUMN, REGION_COLUMN, VAULT_COLUMN, BACKGROUND_COLUMN, ALIGNMENT_COLUMN, BACKSTORY_COLUMN, IMAGE_COLUMN];
     return new Promise((resolve, reject) => {
       let location = {};
-      location[ID_COLUMN] = message.author.id;
+      if (!globe.authorized(message, [globe.roles.MODERATOR])) {
```

Comment:
If this is the only place you added an authorized check for Moderator, I don't believe this resolves #39